### DISCUSSION TITLE
Define Version in Docker Image

### DISCUSSION THREAD

**Discussion by CodyGirl** [grafana/grafana#72244] [Plugins]
How do we define version of plugins while creating a docker image. I tried it like this in the docker file:
`ENV GF_INSTALL_PLUGINS=yesoreyeram-infinity-datasource,grafana-github-datasource,volkovlabs-echarts-panel 4.5.0`

However, it doesn't build and says:
 `Syntax error - can't find = in "4.5.0". Must be of the form: name=value`

Any help would really be appreciated.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
